index.html

```diff
@@ -1,98 +1,165 @@
-                <i>Uncharted 4</i>, <i>God of War</i>, <i>Ghost of Tsushima</i>, <i>Spider-Man</i>, and <i>Horzion Zero Dawn.</i> As of now, the PS4 has sold over 113 million units worldwide.</p>
-            </div>
-            <div id="video-five" class="fancy-video">
-                <h1>Playstation 5</h1>
-                <video src="./videos/PS5.mp4" muted></video>
-                <div class="replay">
-                    <img src="./images/refresh.png" alt="Replay">
-                </div>
-                <p>Sporting the most ostentatious design of all the Playstation consoles, the Playsation 5 released in November 2020. Being led once again by system architect Mark Cerny, the biggest improvement the PS5 toutes is the new solid state drive. With a much faster SSD, loading times
-                    in the video gaming sphere have been dramatically reduced. Internally, the PS5 also includes the first metal cooling in any console as well as ray-tracing on reflective objects in game. The PS5 also received extensive upgrades to the controller as the console series has since ditched the Dualshock series
-                    in favor of the new Dualsense controller. The new controller come equipped with haptic feedback on the controller, a new touchpad, and much more ergonomic style that mimics the larger Xbox controller. With console generations lasting roughly between 6-7 years, the PS5 has a few years for developers
-                    to grow more accustomed to the hardware and take advantage of all the PS5 has to offer.</p>
-            </div>
-            <div> 
-                <p>
-                    <a href="https://www.statista.com/statistics/1101872/unit-sales-video-game-consoles/">Console Sale Projections</a>
-                    <br>
-                    <a href="https://www.lifewire.com/playstation-3-all-you-need-to-know-2717718">What is the Playstation 3?</a>
-                    <br>
-                    <a href="https://www.playstationlifestyle.net/2020/08/14/report-ps5-cooling-solution-liquid-metal-patent/">PS5 Liquid Cooling</a>
-                    <br>
-                    <a href="https://www.britannica.com/topic/PlayStation">Brief Playstation History</a>
-                </p>
-            </div>
-        </main>  
-    </body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>History of Playstation's Consoles</title>
+    <link rel="stylesheet" href="./css/styles.css" />
+    <script src="./js/script.js" defer></script>
+  </head>
+  <body>
+    <nav>
+      <a href="#playstation-one">PS</a>
+      <a href="#playstation-two">PS2</a>
+      <a href="#playstation-three">PS3</a>
+      <a href="#playstation-four">PS4</a>
+      <a href="#playstation-five">PS5</a>
```

Comment:
I renamed the anchors so they made better semantic sense. We're navigating to the sections, not the videos themselves.